database/index.js

```diff
@@ -13,7 +13,7 @@ mongoose.connect(mongoDBURI,
 const db = mongoose.connection;
 
 const bizSchema = mongoose.Schema({
-  bid: Number,
+  bId: Number,
```

Comment:
As an id is added automatically for each collection, adding in bId would mean that there are two different business id's. Perhaps in the future you can limit to the self generated id. 
In my implementation I'm wondering whether to alter the id column to auto increment from 1.